devtools/lightning-inspect.js

```diff
@@ -740,6 +743,21 @@ window.attachInspector = function({Element, ElementCore, Stage, Component, Eleme
         }
     });
 
+    // defining the isfocused property and attribute
+    ElementCore.prototype._isfocused = false;
```

Comment:
Maybe we should camelCase `isFocused` for readability and to follow the pattern of the other properties?

Reply:
You are right!. Done!